libs/payments/cart/src/lib/checkout.service.spec.ts

```diff
@@ -171,18 +170,8 @@ describe('CheckoutService', () => {
           useValue: mockLogger,
         },
         {
-          provide: MozLoggerService,
-          useValue: {
-            error: jest.fn(),
-            debug: jest.fn(),
-            info: jest.fn(),
-            warn: jest.fn(),
-            warning: jest.fn(),
-            log: jest.fn(),
-            verbose: jest.fn(),
-            trace: jest.fn(),
-            setContext: jest.fn(),
-          },
+          provide: LOGGER_PROVIDER,
```

Comment:
L168-L171 👀

Reply:
....................
PG&E did some wild things to our power grid last night at 3am, and its showing